apps/nextjs/src/middlewares/auth.middleware.ts

```diff
@@ -38,26 +47,89 @@ const publicRoutes = [
   "/sign-up(.*)",
 ];
 
-if (process.env.NODE_ENV === "development") {
-  // This allows us to warm up the chat server with live reload
-  // So we get something closer to the live experience
-  // Without having to wait for each page to compile as we navigate
-  publicRoutes.push("/api/chat");
-  publicRoutes.push("/aila");
-}
-
 const isPublicRoute = createRouteMatcher(publicRoutes);
 
+// This allows us to warm up the chat server with live reload
+// So we get something closer to the live experience
+// Without having to wait for each page to compile as we navigate
+const isPreloadableRoute = createRouteMatcher([
+  ...publicRoutes,
+  "/api/chat",
+  "/aila",
+]);
+
+const onboardingUrl = "/onboarding";
```

Comment:
v minor but feels like this doesn't need its own variable?

Reply:
That makes sense. This originated from a clerk example